ocrd_segment/classify_formdata_layout.py

```diff
@@ -383,6 +391,23 @@ def load_image(self, image_id):
         LOG.debug("post-processing time: %d", time4 - time3)
         LOG.debug("contour finding time: %d", time5 - time4)
 
+def tonumpyarray(mp_arr):
+    return np.frombuffer(mp_arr, dtype=np.dtype(mp_arr))
+
+def tonumpyarray_with_shape(mp_arr, shape):
+    return np.frombuffer(mp_arr, dtype=np.dtype(mp_arr)).reshape(shape)
+
+def init(shared_arr_, shape1, second_array, shape2):
+    global shared_arr
+    global shared_shape1 
+    global second_shared_array
+    global shared_shape2 
+    shared_arr = shared_arr_ # must be inherited, not passed as an argument
+    second_shared_array = second_array
+    shared_shape1 = shape1 
+    shared_shape2 = shape2
```

Comment:
I'd prefer more descriptive identifiers:
```suggestion
def morphmasks_init(masks_array, masks_shape, components_array, components_shape):
    global shared_masks
    global shared_masks_shape
    global shared_components
    global shared_components_shape
    shared_masks = masks_array
    shared_masks_shape = masks_shape
    shared_components = components_array
    shared_components_shape = components_shape
```

Reply:
(Should also be moved right above `morphmasks` definition itself.)